liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java

```diff
@@ -107,6 +132,12 @@ public static void main(final String[] args) throws Exception {
         cli.getMethod("main", String[].class).invoke(null, new Object[]{args});
     }
 
+    private static void buildDupsMessage(List<String> duplicates, String title) {
+        StringBuilder builder = new StringBuilder();
+        duplicates.forEach(jar -> builder.append(String.format("%s%n", jar)));
+        Scope.getCurrentScope().getLog(LiquibaseLauncher.class).warning(String.format("*** Duplicate %s JAR files ***%n%s", title, builder));
```

Comment:
Can you output this to the console? I suppose it should also go to the logs, but I definitely think it needs to go to the console so it's obvious.

Reply:
Done